PayPalNativeCheckout/src/main/java/com/braintreepayments/api/PayPalInternalClient.java

```diff
@@ -0,0 +1,131 @@
+package com.braintreepayments.api;
+
+import android.content.Context;
+import android.net.Uri;
+
+import androidx.annotation.Nullable;
+import androidx.annotation.VisibleForTesting;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+
+class PayPalInternalClient {
```

Comment:
The package private classes may collide. I'm not 100% sure, but we should rename to be safe.